packages/snowpack/src/config.ts

```diff
@@ -362,9 +362,21 @@ function loadPlugins(
     const pluginLoc = require.resolve(name, {paths: [process.cwd()]});
     const plugin = require(pluginLoc)(config, options);
     plugin.name = plugin.name || name;
+    // Legacy support: Map the new load() interface to the old build() interface
     if (plugin.build) {
-      plugin.load = (options: LoadOptions) =>
-        plugin.build({...options, contents: fs.readFileSync(options.filePath, 'utf-8')});
+      plugin.load = async (options: LoadOptions) => {
+        const result = await plugin.build({
+          ...options,
+          contents: fs.readFileSync(options.filePath, 'utf-8'),
+        });
+        if (!result) {
+          return null;
+        }
+        if (result.resources) {
+          return {'.js': result.result, '.css': result.resources.css};
+        }
+        return result.result;
+      };
```

Comment:
What are your thoughts on a warning message here? E.g. `This plugin is using version 1. Please update to version 2 here [link]`.

🤔 actually on second thought, that might be annoying. Maybe we should wait for a little bit after releasing to talk about this, then warn after some time has passed.

Reply:
Yea, +1 for waiting a bit.